settings.gradle.kts

```diff
@@ -168,7 +168,7 @@ val groovyBuildScriptProjects = hashSetOf(
 )
 
 fun buildFileNameFor(projectDirName: String) =
-    "$projectDirName${buildFileExtensionFor(projectDirName)}"
+    "build${buildFileExtensionFor(projectDirName)}"
```

Comment:
:x: I think we should remove the `project.buildFileName` and all helper code for that completely, including the the _require_ stuff and the "groovyBuildScriptProjects" list (which only has one entry left anyway). And rely on the default behaviour of Gradle to discover the build scripts.

We should get rid of as much custom setting script build logic as possible.

Reply:
Fixed.